lib/chef-cli/helpers.rb

```diff
@@ -98,6 +98,32 @@ def package_home
                        end
     end
 
+    # Function to return the Chef CLI path based on standalone or Chef-DK-enabled package
+    def get_pkg_install_path
+      # Check Chef-DK package path
+      chef_dk_path = get_pkg_prefix(ChefCLI::Dist::CHEF_DKE_PKG_NAME)
+      return chef_dk_path if chef_dk_path
+
+      # Check Standalone Chef-CLI package path
+      chef_cli_path = fetch_chef_cli_version_pkg || get_pkg_prefix(ChefCLI::Dist::HAB_PKG_NAME)
+      return chef_cli_path if chef_cli_path
```

Comment:
```suggestion
     chef_cli_path
```

Since this is the last line, return is not needed here. Also the chef_cli_path will be either path or nil, so we can just return it.

Reply:
Done